src/core/core.js

```diff
@@ -323,9 +323,6 @@ export default class Core {
         window.performance.mark('yext.answers.verticalQueryResponseRendered');
       }).catch(error => {
```

Comment:
Would it make sense to move this catch immediately after coreLibrary.verticalSearch is called, so that it only catches errors thrown by answers-core? We could then have another catch here at the bottom for all other types of errors

Reply:
per discussion with oshi offline, the throw error should already be clear on where the issue is from. Adding additional catch statements here would require handling breaking the then/catch chain mid way or add logic to avoid bleeding into the later catch statements (potentially end up with duplicate update loading state calls and console errors)